### PR TITLE
Replace buttons & inputs for blade-components | Added translations

### PR DESCRIPTION
**Pull request by Roene-JustBetter**
**This is for V3.** 

- Replace the inputs & buttons for the new blade-components. 
- Make use of new colors.
- Added translations + workflow.

### CODE REVIEW

**Comment by royduin** [comment]
Waiting for at least a good review from @jordythevulder or @JimmyHoenderdaal

**Review by jordythevulder — APPROVED** [review]

**Pull request merged**
